core/src/main/scala/com/avast/clients/storage/storage.scala

```diff
@@ -0,0 +1,67 @@
+package com.avast.clients
+
+import better.files.File
+import cats.data.EitherT
+import cats.{~>, Monad}
+import com.avast.scala.hashes.Sha256
+import mainecoon.FunctorK
+
+import scala.language.higherKinds
+
+package object storage {
+
+  implicit class StorageClientOps[F[_]](val client: StorageClient[F]) extends AnyVal {
+    def mapK[G[_]](implicit fToG: F ~> G): StorageClient[G] = {
+      storageClientFunctorK.mapK(client)(fToG)
+    }
+  }
+
+  implicit class StorageBackendOps[F[_]: Monad](val backend: StorageBackend[F]) {
```

Comment:
`extends AnyVal`?

Reply:
Nope. The fact that it has `[F[_]: Monad]` causes that there is second param (`implicit val sth: Monad[F]` generated by compiler) thus it could not `extend AnyVal`.